### PR TITLE
NIAD-957: Add CL tools and instructions to communicate with fake mesh

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
## Description

Copied and tweaked from NHAIS project. Tested with

```
    cd mesh
    ./mesh.sh send gp_mailbox "@../src/intTest/resources/edifact/registration.dat"
```

fake-mesh docker image reports:

>    POST https://localhost:8829/messageexchange/gp_mailbox/outbox -> 202 ACCEPTED

Then, when the next database lock is obtained by lab-results docker image:

>     INFO u.n.d.n.lab.results.mesh.MeshService  There are 1 messages in the MESH mailbox

and it proceeds to process the message as normal.

## Jira Ticket

https://gpitbjss.atlassian.net/browse/NIAD-957

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [x] Acceptance Criteria met
- [x] Commit messages are meaningful
- [x] Manually tested
- [x] Self-reviewed your code
- [x] Code reviewed from two other developers
- [x] If your pull request depends on any other, please link them in the description
- [x] main branch is passing/green on CI
- [x] Add/update any relevant documentation

### CODE REVIEW

File: mesh/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Use xargs to run in parallel
+
+     $ time echo {0..1000} | xargs -n 1 -P 4 ./mesh.sh send @GPHA_SAMPLE.dat
+       echo {0..1000}  0.00s user 0.00s system 63% cpu 0.001 total
+       xargs -n 1 -P 4 ./mesh.sh send @GPHA_SAMPLE.dat  35.40s user 18.87s system 164% cpu 32.905 total
+
+Shows 1000 / 32.905 ~= 30 writes per second using 4 parallel processes. Note that the fake-mesh database 
+(LMDB) does not support parallel writes.
+
+Setting the option lock=False in the database driver doubles throughput, but I'm unsure if this is a good idea.
```
Comment (andrewwa-kainos) [inline mesh/README.md:77]: I'm inclined to remove the benchmarking section as we have no requirement for it *yet*, and (for that reason) I haven't tested it.

Comment (n-r-b) [reply mesh/README.md:77]: It's not relevant right now, so i would agree with removing this

**Review by n-r-b — APPROVED** [review]
Tested locally - LGTM!

**Review by saileshp-kainos — APPROVED** [review]
LGTM 😄

**Pull request merged**